src/Vizkit3dWorld.cpp

```diff
@@ -155,6 +164,14 @@ vizkit3d::RobotVisualization* Vizkit3dWorld::robotVizFromSdfModel(sdf::ElementPt
     //vizkit3d plugin with model defined in the sdf model
     vizkit3d::RobotVisualization* robotViz = new vizkit3d::RobotVisualization();
 
+    for (int i=0; i < excludeFromSdf.size(); ++i){
```

Comment:
One thing I just realized: why do you exclude the visual/collision links instead of ignoring the model ?